spec/requests/topics_controller_spec.rb

```diff
@@ -2322,6 +2322,12 @@ def fabricate_topic(user, category = nil)
       expect(response).to redirect_to(topic.relative_url)
     end
 
+    it "does not raise an unhandled exception when receiving an array of IDs" do
+      get "/t/#{topic.id}/summary?id[]=a,b"
```

Comment:
Example of a route that hits this endpoint, but doesn't have an `:id` fragment. (It's named `:topic_id` in this route.)